gradle/maven.gradle

```diff
@@ -60,12 +60,21 @@ plugins.withId('maven') {
                     }
                 }
 
```

Comment:
we need to be moving to "maven-publish" plugin from Gradle 7 onwards right?(dont see that)

Reply:
Yes, there is. If you use version gradle is greater or equal than 7.0 then you need uncomment the code:

![image](https://user-images.githubusercontent.com/86481341/211049188-8a325a67-1b1f-4dfb-b921-7a09dc8c9bd1.png)